README.md

```diff
@@ -56,51 +51,88 @@ function Worker() {
   - `trustComputeGroupOrigins?: boolean = true`: Trust the scheduler to tell client about allowed origins for jobs in a compute group.
-  - `allowOrigins?: object`: Allow list permitting network access beyond DCP messages to services.
-    - `any: []`: A list of origins that are safe to communicate with.
-    - `fetchWorkFunctions: []`: A list of work function URIs that are safe to communicate with.
-    - `fetchArguments: []`: A list of argument datum URIs that are safe to communicate with.
-    - `fetchData: []`: A list of input datum URIs that are safe to communicate with.
-    - `sendResults: []`: A list of URIs that are safe to send job results to.
+  - `allowOrigins?: object`: Allow list permitting network access beyond DCP messages to services. This list is used only in setting up the DCP Worker. After the worker is constructed/loaded, the `originManager` is responsible for managing origins (see Managing Origins). Empty by default of course.
+    - `any: []`: A list of origins that are allowed to communicate with, for all purposes.
+    - `fetchWorkFunctions: []`: A list of origins that are allowed to fetch the work function from.
+    - `fetchArguments: []`: A list of origins that are allowed to fetch work function arguments from.
+    - `fetchData: []`: A list of origins that are allowed to fetch input set data from.
+    - `sendResults: []`: A list of origins that are allowed to send results to.
   - `minimumWage?: object`: The minimum payout per slice the worker will accept from a job. Will default with the following structure:
     - `CPU: number = 0`
     - `GPU: number = 0`
     - `in: number = 0`
     - `out: number = 0`
   - `computeGroups?: []`: List of compute groups the worker is in and the authorization to join them. A compute group is to be described as `{ joinKey: 'exampleGroup', joinSecret: 'password' }`.
+  - `leavePublicGroup?: boolean = false`: A flag that controls if the worker should omit fetching work from the public compute group. If not defined, this flag is evaluated to _false_.
   - `jobAddresses?: []`: If populated, worker will only fetch slices from jobs corresponding to the job addresses in this list.
   - `maxWorkingSandboxes?: integer | undefined`: Maximum number of sandboxes allowed to do work. If `undefined`, then the Supervisor will determine a safe limit, based off of machine hardware.
-  - `paymentAddress: Keystore | Address | String`: A Keystore or Address (`dcp.wallet.Address`) identifying a DCP Bank Account to deposit earned DCCs. An address string can also be supplied.
+  - `paymentAddress: Keystore | Address | String`: A Keystore, Address (`dcp.wallet.Address`) or string identifying a DCP Bank Account to deposit earned DCCs.
```

Comment:
Ive updated it so its: string | Address | Keystore. I believe string is preferable over String

Reply:
https://github.com/Distributive-Network/use-dcp-worker/pull/2#discussion_r1293642571

> I'd also like to fully address the **first half** of https://github.com/Distributive-Network/use-dcp-worker/pull/2#discussion_r1245730062 as well..

My bad, I wasn't being clear. I was referring to this part of the suggestion:

> suggestion: I think the mandatory property should be listed first so that folks looking for the bare minimum can more easily parse it from skimming the text.

i.e., Order `paymentAddress` first in the docs, rather than near the end/mixed in with the other parameters.

**question:** Should I split my suggestions in to multiple comments to avoid this confusion in the future?

Reply:
> Should I split my suggestions in to multiple comments to help avoid this confusion in the future?
No this is my fault I had misinterpreted what your comment was - Sorry! Ill make this change.